apps/web/ui/links/short-link-input-new.tsx

```diff
@@ -439,3 +403,74 @@ function DefaultDomainPrompt({
     </button>
   );
 }
+
+function DomainCombobox({
+  domain,
+  setDomain,
+  loading,
+}: {
+  domain?: string;
+  setDomain: (domain: string) => void;
+  loading: boolean;
+}) {
+  const [search, setSearch] = useState("");
+  const [debouncedSearch] = useDebounce(search, 500);
+  const { domains, loading: loadingDomains } = useAvailableDomains({
+    search: debouncedSearch,
+  });
```

Comment:
No need for passing a `currentDomain` anymore because the Combobox handles displaying the current selection even if it's not in the `options` array